src/xcode/ENA/ENA/Source/Services/CCLService/DCCWalletInfo.swift

```diff
@@ -5,10 +5,19 @@
 import Foundation
 import AnyCodable
 
+enum MaskStateIdentifier: String, Codable {
+
+	case maskRequired = "MASK_REQUIRED"
+	case maskOptional = "MASK_OPTIONAL"
+	case other = "OTHER"
+
```

Comment:
please remove the extra space

Reply:
It's fine we have this structure in entire file